MediaBrowser.Model/Lyrics/LyricLineTimeTag.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Model.Lyrics;
+
+/// <summary>
+/// LyricLineTimeTag model.
+/// </summary>
+public class LyricLineTimeTag
```

Comment:
Would make sense to use "Cue" instead of "Tag" for consistency with the property name in LyricLine

```suggestion
public class LyricLineCue
```